ios/FluentUI/Bottom Sheet/BottomSheetView.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class BottomSheetView: UIView {
+    public override init(frame: CGRect) {
+        super.init(frame: frame)
+
+        // We need to have the shadow on a parent of the view that does the corner masking.
+        // Otherwise the view will mask its own shadow.
+        layer.shadowColor = Constants.Shadow.color
+        layer.shadowOffset = Constants.Shadow.offset
+        layer.shadowOpacity = Constants.Shadow.opacity
+        layer.shadowRadius = Constants.Shadow.radius
+
+        addSubview(contentView)
+
+        NSLayoutConstraint.activate([
+            contentView.leadingAnchor.constraint(equalTo: leadingAnchor),
+            contentView.trailingAnchor.constraint(equalTo: trailingAnchor),
+            contentView.topAnchor.constraint(equalTo: topAnchor),
+            contentView.bottomAnchor.constraint(equalTo: bottomAnchor)
+        ])
+    }
+
+    required init?(coder: NSCoder) {
+        preconditionFailure("init(coder:) has not been implemented")
+    }
+
+    private(set) lazy var contentView: UIView = {
+        let contentView = UIView()
+        contentView.backgroundColor = Colors.NavigationBar.background
+        contentView.translatesAutoresizingMaskIntoConstraints = false
+        contentView.layer.cornerRadius = Constants.cornerRadius
+        contentView.layer.cornerCurve = .continuous
+        contentView.layer.maskedCorners = [.layerMaxXMinYCorner, .layerMinXMinYCorner]
+        contentView.clipsToBounds = true
+        return contentView
+    }()
+    
+    private struct Constants {
+        static let cornerRadius: CGFloat = 14
+
+        struct Shadow {
+            static let color: CGColor = UIColor.black.cgColor
+            static let opacity: Float = 0.14
+            static let radius: CGFloat = 8
+            static let offset: CGSize = CGSize(width: 0, height: 4)
+        }
+    }
```

Comment:
I think we've moved away from structs for constants and just call em private vars.

Reply:
Do you know of a recent example? Everywhere I look it's this. And I kind of like this style 😄

Reply:
Talked offline, but ironically I couldn't find an example in the first two files I mentioned (FluentUI Apple macOS Button and Link). I guess we're bad at following it :D 

The Constants Struct is something we inherited before we settled on the ordering in aka.ms/swift-guide , for context of anyone reading this